docs/SPEC.md

```diff
@@ -2146,6 +2146,69 @@ Examples:
     | 0002  | "temp" | 75 | "temp" | 56 |
     | 0003  | "temp" | 72 | "temp: | 55 |
 
+#### Union
+
+Union concatenates two input streams into a single output stream.  In tables that have identical
+schema and group keys, contents of the tables will be concatenated in the output stream.
+
```

Comment:
Can you list the properties (or in this case property) of Union under this description?